pandas/core/internals/concat.py

```diff
@@ -49,19 +90,7 @@ def concatenate_block_managers(
     BlockManager
     """
     if isinstance(mgrs_indexers[0][0], ArrayManager):
```

Comment:
are we implicitly assuming all-ArrayManager here?

Reply:
Yes, that's an open TODO, will add a comment

Reply:
thanks